### DISCUSSION TITLE
Feature Request: Keyboard shortcut to playback/recording (sounds)

### DISCUSSION THREAD

**Discussion by Makki10** [File-New-Project/EarTrumpet#683] [Ideas]
In a earlier release of EarTrumpet there was a shortcut for "sounds" that was not behind a dropdown, which was amazing. Would be even more amazing with a keyboard shortcut though.

Absolutely love EarTrumpet and thank you for your passion and time.

**Comment by AngryLifeender** [comment]
Yes please, Keyboard shortcut to open Windows Legacy "Playback Devices" and "Recording Devices". 
And they need to change the placement for "Playback devices" and "Recording Devices" back to the first right-click menu instead of having to hover over "Windows Legacy".
